hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/MergeOnReadIncrementalRelation.scala

```diff
@@ -119,6 +120,35 @@ case class MergeOnReadIncrementalRelation(override val sqlContext: SQLContext,
     }
   }
 
+  def listFileSplits(partitionFilters: Seq[Expression], dataFilters: Seq[Expression]): Map[InternalRow, Seq[FileSlice]] = {
```

Comment:
Could this be extracted out as a util method instead of sitting inside the MOR incremental relation, which will not be used by the new Hudi parquet file format class?

Reply:
This function is created to extract the file splits from the relation, and will not be used by the file format.